### DISCUSSION TITLE
Customized Command bar in Edit page

### DISCUSSION THREAD

**Discussion by LorenzoAlu** [orchidsoftware/platform#1867] [Q&A]
Hi everybody.

I am using `orchidsoftware/CRUD` package and it's awesome. 
I have a question. 

Is there a way to customize the defaults command (Update, Delete model) in edit page?

let me explain.
I want do to enable/disable the delete default button based on a certain condition,
I can do that without create custom layout and screen?

Also I see that you can use, in your resource controller, `onDelete()` and `onSave()` methods, but is there any way to control directly the `delete()` and 'save()' action?
I would like make some check before `delete()` action and throw an exception, that I can see in frontend, in case of occurs errors.

Thank you

**Comment by tabuna** [comment]
You can disable it using the model policies https://github.com/orchidsoftware/crud#policies

**Reply by LorenzoAlu** [reply]
Your're the best my friend

thank you soooo much ;)
